Client.Linq/Internal/Expressions/String/ToString.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using System.Text;
+
+namespace InfluxDB.Client.Linq.Internal.Expressions.String
+{
+    internal class ToString : IExpressionPart
+    {
+        private readonly IEnumerable<IExpressionPart> expressionParts;
+
+        internal ToString(IEnumerable<IExpressionPart> expressionParts)
+        {
+            this.expressionParts = expressionParts;
+        }
+
+        public void AppendFlux(StringBuilder builder)
+        {
+            builder.Append("string(v: ");
+            foreach (var expressionPart in expressionParts) expressionPart.AppendFlux(builder);
+            builder.Append(")");
+        }
```

Comment:
Please use same naming as is in Flux documentation: https://docs.influxdata.com/flux/v0.x/stdlib/universe/string/